cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsInstanceConnector.java

```diff
@@ -55,6 +55,11 @@ public class AwsInstanceConnector implements InstanceConnector {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AwsInstanceConnector.class);
 
+    private static Collection<String> pendingInstances;
```

Comment:
SHould this be static or can it be passed as argument to the relevant method directly?

Reply:
Okay, we can declare it as non-static and pass it to relevant method only.